src/main/content/antora_ui/src/css/doc.css

```diff
@@ -591,7 +591,7 @@ kbd,
   height: 100%;
 }
 
-.doc .paragraph p {
+.noversion-paragraph {
```

Comment:
the .doc .paragraph p { is being use every where in the docs, just want to make sure we are not regress the behavior

Reply:
previously I had added css to .doc .paragraph p { which effects everywhere, that was causing this font issue https://openliberty.io/docs/latest/network-hardening.html#_mutual_authentication

now I am adding and removing class specifically in no-version page, so can see the difference its not effects everywhere, its applies only in no-version page
https://demo1-openlibertyio.mybluemix.net/docs/latest/network-hardening.html#_mutual_authentication

Reply:
lgtm